projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-as-persistent-user.ts

```diff
@@ -187,22 +215,46 @@ export function selectDeliveryMethod() {
     'GET',
     '/rest/v2/electronics-spa/cms/pages?*/checkout/payment-details*'
   ).as('getPaymentPage');
-  cy.get('.cx-checkout-title').should('contain', 'Shipping Method');
+
+  cy.get('cx-checkout-progress')
+    .should('exist')
+    .then(() => {
+      cy.get('cx-checkout-progress a.is-active').should(
+        'contain',
+        '2. Delivery mode'
```

Comment:
I don't know what to say for this e2e anymore :( It fails here still. Not your fault but I guess the core of cypress? or maybe our application?? no idea (we need ideas from others I believe), but look above

I started running the builds at 313-317 (you can see 312 is different since I have no idea who started that one), and it failed at 313